manim/mobject/svg/text_mobject.py

```diff
@@ -1056,11 +1062,454 @@ def text2svg(self):
+                )
+                text7 = MarkupText(
+                    'fl ligature <color col="GREEN" offset="1,1">floating</color> inside'
+                )
+                group = VGroup(text1, text2, text3, text4, text5, text6, text7).arrange(DOWN)
+                self.add(group)
+
+    .. manim:: FontExample
+        :save_last_frame:
+
+        class FontExample(Scene):
+            def construct(self):
+                text1 = MarkupText(
+                    'all in sans <span font_family="serif">except this</span>', font="sans"
+                )
+                text2 = MarkupText(
+                    '<span font_family="serif">mixing</span> <span font_family="sans">fonts</span> <span font_family="monospace">is ugly</span>'
+                )
+                text3 = MarkupText("special char > or &gt;")
+                text4 = MarkupText("special char &lt; and &amp;")
+                group = VGroup(text1, text2, text3, text4).arrange(DOWN)
+                self.add(group)
+
+    .. manim:: NewlineExample
+        :save_last_frame:
+
+        class NewlineExample(Scene):
+            def construct(self):
+                text = MarkupText('foooo<color col="RED">oo\nbaa</color>aar')
+                self.add(text)
+
+    .. manim:: NoLigaturesExample
+        :save_last_frame:
+
+        class NoLigaturesExample(Scene):
+            def construct(self):
+                text1 = MarkupText('fl<color col="RED">oat</color>ing')
+                text2 = MarkupText('fl<color col="RED">oat</color>ing', disable_ligatures=True)
+                group = VGroup(text1, text2).arrange(DOWN)
+                self.add(group)
+
+
+    As :class:`MarkupText` uses Pango to render text, rendering non-English
+    characters is easily possible:
+
+    .. manim:: MultiLanguage
+        :save_last_frame:
+
+        class MultiLanguage(Scene):
+            def construct(self):
+                morning = MarkupText("வணக்கம்", font="sans-serif")
+                chin = MarkupText(
+                    '見 角 言 谷  辛 <color col="BLUE">辰 辵 邑</color> 酉 釆 里!'
+                )  # works as in ``Text``.
+                mess = MarkupText("Multi-Language", style=BOLD)
+                russ = MarkupText("Здравствуйте मस नम म ", font="sans-serif")
+                hin = MarkupText("नमस्ते", font="sans-serif")
+                japanese = MarkupText("臂猿「黛比」帶著孩子", font="sans-serif")
+                group = VGroup(morning, chin, mess, russ, hin, japanese).arrange(DOWN)
+                self.add(group)
+
+
+    Tests
+    -----
+
+    Check that the creation of :class:`~.MarkupText` works::
+
+        >>> MarkupText('The horse does not eat cucumber salad.')
+        MarkupText('The horse does not eat cucumber salad.')
+
+    .. WARNING::
+
+        Using a :class:`.Transform` on text with leading whitespace can look
+        `weird <https://github.com/3b1b/manim/issues/1067>`_. Consider using
+        :meth:`remove_invisible_chars` to resolve this issue.
```

Comment:
Is this warning still valid?

Reply:
Good question; I honestly do not know. Is it still valid for the classical `Text` class? I have never encountered the problem myself, so I did not want to remove that warning.

Reply:
I also just copied from the previous `Text` class. Maybe remove it everywhere where pango is in?

Reply:
I just tested it with `Text` and `MarkupText`, using the test case described in the linked 3b1b PR. I do not see anything wierd, so I removed the warning. I have left it for `CairoText` although I do not see any problem there either. We might want to remove it altogether, but I'd rather not decide that just on my own without further testing.

Reply:
I've also just adopted this warning from the previous docstring; to be honest I never really understood why this was pointed out so explicitly everywhere...

Reply:
So I removed it now for all classes.